MultiTarget/AvailableTargetFrameworks.props

```diff
@@ -1,23 +1,23 @@
 <Project>
     <PropertyGroup>
-        <UwpTargetFramework>uap10.0.17763</UwpTargetFramework>
-        <WinAppSdkTargetFramework>net8.0-windows10.0.22621.0;net7.0-windows10.0.22621.0;net6.0-windows10.0.22621.0;</WinAppSdkTargetFramework>
+        <UwpTargetFramework Condition="'$(UwpTargetFramework)' == ''">uap10.0.17763</UwpTargetFramework>
+        <WinAppSdkTargetFramework Condition="'$(WinAppSdkTargetFramework)' == ''">net8.0-windows10.0.22621.0;net7.0-windows10.0.22621.0;net6.0-windows10.0.22621.0;</WinAppSdkTargetFramework>
         
-        <WasmHeadTargetFramework>net8.0</WasmHeadTargetFramework>
-        <LinuxHeadTargetFramework>net8.0</LinuxHeadTargetFramework>
-        <WpfHeadTargetFramework>net8.0</WpfHeadTargetFramework>
+        <WasmHeadTargetFramework Condition="'$(WasmHeadTargetFramework)' == ''">net8.0</WasmHeadTargetFramework>
```

Comment:
Not sure I understand the point of this:
- If these .props are imported before the .csproj, how are they ever going to see the .csproj overrides? And the .csproj being imported after them will still override the value anyway.
- If these .props are imported after the .csproj, why are they called .props and not .targets? 😅

Reply:
They're imported inside the sample or source csproj, first through [ToolkitComponent.SourceProject.props](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/blob/main/ToolkitComponent.SourceProject.props) or [ToolkitComponent.SourceProject.props](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/blob/main/ToolkitComponent.SampleProject.props), which references [./MultiTarget/Library.props](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/blob/main/MultiTarget/Library.props) to pick these up.

Reply:
Gotcha, ok so they're just called .props but they're effectively .targets, I get it now 😄